Majasdarbs/05-majasdarbs/Object/cv.js

```diff
@@ -1,3 +1,21 @@
 const VardsUzvards = {
+    name: "Stanislavs"
+    surname: "Lopatinskis"
+    email: "stanislavs.lopatinskis@gmail.com"
+    education: {
+        "Politikas zinātnes maģistrs 2011-2014"
+        "Politikas zinātnes bakalaurs 2008-2011"
+    }
+    skills: {
+        "Adobe Lightroom"
+        "Adobe Photoshop"
+        "Google Optimize"
+    }
+    languages: {
+        "Latvian"
```

Comment:
key should have value.
in these cases it should be
languages: ["Latvian", "Russian", etc.]

Similar in other places